crates/pop-common/src/lib.rs

```diff
@@ -3,17 +3,26 @@ pub mod errors;
 pub mod git;
 pub mod helpers;
 pub mod manifest;
+pub mod metadata;
 pub mod polkadot_sdk;
+pub mod signer;
```

Comment:
Docs for public modules

Reply:
Good catch. Added for the ones introduced in this PR (https://github.com/r0gue-io/pop-cli/pull/316/commits/44c1d4ebd330320e6e2efba183aeebbe28cb51ec).
But as above for existing ones, I feel this should be addressed in a separate PR, as this one is already a big one and adding unrelated changes would make it more complex to review.

Reply:
Could we maybe make an issue for this?

Reply:
Yes, we should not forget: https://github.com/r0gue-io/pop-cli/issues/364